h/groups/templates/share.html.jinja2

```diff
@@ -18,6 +18,18 @@
       <div class="group-form__name-input">
         {{ group.name }}
       </div>
+
+      {% if document_links %}
+      Recently annotated documents:
+      <ul>
+        {% for document_link in document_links %}
+          <li>{{ document_link }}</li>
+        {% endfor %}
+      </ul>
+      {% else %}
+        <p>This group hasn't shared any annotations yet.</p>
```

Comment:
I think I prefer my wording as it's simpler, and I think users will understand "the group" to mean both the group (i.e. the thing that exists in our database) and the users who're in that group (this seems like typical usage of the word group to me). I just prefer direct language, I'm sick of seeing needlessly indirect things like _this annotation is viewable only by you_ instead of _only you can see this annotation_ etc etc in Hypothesis.